src/redux/books/books.js

```diff
@@ -1,50 +1,97 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
 import { v4 as uuid } from 'uuid';
-import initialbook from './initialbooks';
+import axios from 'axios';
 
-const storedBooks = () => {
-  const storedBooks = localStorage.getItem('Allbooks');
-  const storedBooksArr = JSON.parse(storedBooks);
+const initialState = {
+  books: [],
+  isLoading: false,
+};
 
-  if (storedBooks) {
-    if (storedBooksArr.length !== 0) {
-      return storedBooksArr;
-    }
-    return initialbook;
+export const api = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/7kYFUWenXnjBJNyIFqkh/books';
```

Comment:
- [OPTIONAL ] You did a great work to implement API with redux, but I am not sure if its due to the api you are using, but the loading speed of your website if very low. If you could find a way of improve the loading speed that would be amazing, and your website will be super cool.